web/src/components/LanguageList.tsx

```diff
@@ -0,0 +1,160 @@
+import Autocomplete from '@mui/material/Autocomplete'
+import Box from '@mui/material/Box'
+import List from '@mui/material/List'
+import TextField from '@mui/material/TextField'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { normalizeString } from 'shared'
+import { config } from 'translations'
+
+import useDimensions from '../hooks/useDimensions'
+import LanguageListItem from './LanguageListItem'
+import SearchInput from './SearchInput'
+
+const MobileContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
🔧 i don't think height and width are needed here? I didn't find any issue without it